salty-747/html_ui/Pages/VCockpit/Instruments/Airliners/B747_8/FMC/B747_8_FMC_ProgPage.js

```diff
@@ -145,22 +148,59 @@ class B747_8_FMC_ProgPage {
         else {
             crzSpeedCell = Simplane.getAutoPilotAirspeedHoldValue().toFixed(0);
         }
+        let todTimeCell = "";
+        let todDistanceCell = "";
+        let toToDText = "";
+        let distanceToTOD = SimVar.GetSimVarValue("L:WT_CJ4_TOD_REMAINING", "number");
+        if (distanceToTOD > 0) {
+            if (isFinite(distanceToTOD) && distanceToTOD < 201) {
+                toToDText = "TO T/D";
+                if (distanceToTOD < 5) {
+                    todDistanceCell = "NOW";
```

Comment:
Looking into this further, the NOW does not display until the TOD point has already passed, there is no 5NM buffer. So some small value such as >0.1nm would be better.

Reply:
How should this looks like?